src/terra-dev-site/IssueForm/FeatureForm.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Field } from 'react-final-form';
+import TextareaField from 'terra-form-textarea/lib/TextareaField';
+import { propTypes, validateForm } from './Helper';
+
+const FeatureForm = (props) => {
+  const {
+    context,
+    mentions,
+    setContext,
+    setDescription,
+    setMentions,
+    setTitle,
+  } = props;
+
+  return (
+    <div>
+      <Field
+        name="title"
+        validate={validateForm}
+      >
+        {({ input, meta }) => (
+          <TextareaField
+            label="Title"
+            aria-label="Brief description. Selected package will be included in title."
+            inputId="title"
+            required
+            error={meta.error}
+            isInvalid={meta.invalid && meta.touched}
+            help="Brief description. Selected package will be included in title."
+            maxWidth="50em"
+            onChange={(event) => { input.onChange(event.target.value); setTitle(event.target.value); }}
+            value={input.value}
+            inputAttrs={{ ...input }}
+          />
+        )}
+      </Field>
+      <Field
+        name="description"
+        validate={validateForm}
+      >
+        {({ input, meta }) => (
+          <TextareaField
+            label="Description"
+            aria-label="A clear and concise description of what the feature is."
+            inputId="description"
+            required
+            error={meta.error}
+            isInvalid={meta.invalid && meta.touched}
+            help="A clear and concise description of what the feature is."
+            maxWidth="50em"
+            onChange={(event) => { input.onChange(event.target.value); setDescription(event.target.value); }}
+            value={input.value}
+            inputAttrs={{ ...input }}
+          />
+        )}
+      </Field>
+      <TextareaField
+        label="Additional Context / Screenshots"
+        aria-label="Add any other context about the feature here. If applicable, add screenshots to help explain."
+        inputId="context"
+        help="Add any other context about the feature here. If applicable, add screenshots to help explain."
+        maxWidth="50em"
+        value={context}
+        onChange={event => setContext(event.target.value)}
+      />
+      <TextareaField
+        label="Mentions"
+        aria-label="@ Mention anyone on the terra team that you have been working with so far."
+        inputId="mentions"
+        help="@ Mention anyone on the terra team that you have been working with so far."
```

Comment:
Let's remove the `@` character from here. GitHub issue form has a feature to auto-suggest usernames when you type `@` on the form. I'm not too interested in replicating that here and think most people won't know our GitHub usernames so it would be eaiser to understand if they just type in our names here.

Reply:
Updated: [d658958](https://github.com/cerner/terra-ui/pull/151/commits/d6589587b241e85489f9f156c29d1dab408f90d0)